### PR TITLE
WIP -- Break out fixtures and beaker rake tasks into sperate files

### PR DESCRIPTION
**Pull request by cyberious**
- More changes coming but this is the start of a refacter and enhancement

### CODE REVIEW

**Comment by DavidS** [comment]
That looks like a good cleanup. I heard @pinkypie was looking into a similar project on the beaker side. It would be great if we could get everyone on the same page here.

File: lib/puppetlabs_spec_helper/rake_tasks.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 require 'rake'
+require 'beaker/rake_tasks'
+require 'fixtures/rake_tasks'
```
Comment (DavidS) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: shouldn't these both be `puppetlabs_spec_helper/...` ?

Comment (cyberious) [inline lib/puppetlabs_spec_helper/rake_tasks.rb]: It is a relative base and should not require the full namespace.

**Comment by eputnam** [comment]
@cyberious are you planning to continue this work?

**Comment by DavidS** [comment]
Superseded by #238

**Pull request closed without merging**